infer_structcol/model.py

```diff
@@ -84,34 +88,46 @@ def calc_resid_spect(spect1, spect2):
                     sigma_r = sigma_eff_r, transmittance = residual_t, 
                     sigma_t = sigma_eff_t)
 
-def calc_log_prior(theta):
+def calc_log_prior(theta, theta_range):
```

Comment:
Moved the theta_guess_default and theta_range_default to inference.py since they aren't used in model.py anymore